### PR TITLE
Feature/match orders

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: lib/iexecbroker/core.ts [hunk 1]
```diff
@@ -138,6 +138,68 @@ export default class Core extends IexecOrderFetcher
...
 	}
 
+    async tryMatchOrders(brokerOrder: types.BrokerOrder) : Promise<types.DealDescriptor>
+    {
+        let context: string = `[requester:${brokerOrder.requestorder.requester}, ` +
+            `app:${brokerOrder.requestorder.app}, ` +
+            `workerpool:${brokerOrder.requestorder.workerpool}, ` +
+            `dataset:${brokerOrder.requestorder.dataset}]`
+
+        console.log(`INFO: tryMatch ${context}`);
```
[the remark this answers is not in the document]
Comment (jeremyjams) [reply lib/iexecbroker/core.ts]: Updated thanks

File: lib/iexecbroker/core.ts [hunk 2]
```diff
@@ -138,6 +138,68 @@ export default class Core extends IexecOrderFetcher
...
+    }>
+    {
+        let match:           types.DealDescriptor = undefined;;
+        let error:           string                 = undefined;
+        let brokerOrder:     types.BrokerOrder     = types.toBrokerOrder(raw);
+
+        let context: string = `[requester:${brokerOrder.requestorder.requester}, ` +
+            `app:${brokerOrder.requestorder.app}, ` +
+            `workerpool:${brokerOrder.requestorder.workerpool}, ` +
+            `dataset:${brokerOrder.requestorder.dataset}]`
```
[the remark this answers is not in the document]
Comment (jeremyjams) [reply lib/iexecbroker/core.ts:175]: Yes, added appropriate requires thanks

File: lib/iexecbroker/iexec-order-fetcher.ts [hunk 3]
```diff
@@ -4,6 +4,9 @@ import * as types       from './utils/types';
 
 const randomChoice = items => items[items.length * Math.random() | 0];
 
+const chain:       string        = process.env.CHAIN || 'goerli';
+const address:     string        = process.env.PROXY || 'core.v5.iexec.eth';
```
[the remark this answers is not in the document]
Comment (jeremyjams) [reply lib/iexecbroker/iexec-order-fetcher.ts]: Updated :)

**Review by zguesmi — APPROVED** [review]
Topissimo!

**Pull request merged**